env-rendering/config_renderer.py

```diff
@@ -0,0 +1,12 @@
+import yaml
+import os
+
+vals = dict()
+
+try:
+    vals['MAGIC'] = os.environ['MAGIC']
+except KeyError:
+    vals['MAGIC'] = 'Magic val not found in the environment variables'
+
+with open('/etc/app/config.yml', 'w') as f:
```

Comment:
Ummm, how about this instead? https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/#populate-a-volume-with-data-stored-in-a-configmap